ui/src/data/containers/DetailPageContainer/DetailPageContainer.tsx

```diff
@@ -90,11 +90,7 @@ const DetailPage = ({
                 </div>
               </Col>
             </Row>
-            {dois && (
-              <Row>
-              <Col><DOIList dois={dois} /></Col>
-            </Row>
-            )}
+            <DOIListShowAll dois={allDois} dataDois={dataDois} hasOtherDois={hasOtherDois} />
```

Comment:
I would import and call `hasAdditionalDois` and `filterDoisByMaterial` directly inside `DOIListShowAll` instead of passing them here as props (pass only `dois`), as we're not using them in this component. It is okay though, just my two cents :)

Reply:
Indeed! Will check this too!